azure-bundle/LICENSE

```diff
@@ -511,9 +511,3 @@ License: The Apache Software License, Version 2.0 - http://www.apache.org/licens
 Group: org.reactivestreams  Name: reactive-streams  Version: 1.0.4
 Project URL: http://www.reactive-streams.org/
 License: MIT-0 - https://spdx.org/licenses/MIT-0.html
-
---------------------------------------------------------------------------------
-
-Group: org.slf4j  Name: slf4j-api  Version: 1.7.36
```

Comment:
```
➜ cp /Users/fokko.driesprong/work/iceberg/azure-bundle/build/libs/iceberg-azure-bundle-1.8.0-SNAPSHOT.jar .
➜ unzip iceberg-azure-bundle-1.8.0-SNAPSHOT.jar
➜ find . | grep -i slf                                                                                     
./org/apache/iceberg/azure/shaded/io/netty/util/internal/logging/LocationAwareSlf4JLogger.class
./org/apache/iceberg/azure/shaded/io/netty/util/internal/logging/Slf4JLoggerFactory$NopInstanceHolder.class
./org/apache/iceberg/azure/shaded/io/netty/util/internal/logging/Slf4JLoggerFactory.class
./org/apache/iceberg/azure/shaded/io/netty/util/internal/logging/Slf4JLogger.class
./reactor/util/Loggers$Slf4JLoggerFactory.class
./reactor/util/Loggers$Slf4JLogger.class
```

Reply:
I don't think we can consider https://github.com/netty/netty/blob/4.1/common/src/main/java/io/netty/util/internal/logging/Slf4JLogger.java as slf4j-api provided. It's different package/class. 

The Netty `NOTICE` (https://github.com/netty/netty/blob/4.1/NOTICE.txt) mentions slf4j:

```
This product contains a modified portion of 'SLF4J', a simple logging
facade for Java, which can be obtained at:

  * LICENSE:
    * license/LICENSE.slf4j.txt (MIT License)
  * HOMEPAGE:
    * https://www.slf4j.org/

```
That should be in our `NOTICE` (I will update the `NOTICE` in this PR).

So, strictly speaking, the Slf4jLogger is not provided by slf4j-api.

I still think that we should remove slf4-api from `LICENSE` because `org.slf4j*` is not bundle.